views/templates/layout.html.twig

```diff
@@ -42,5 +42,6 @@
       </div>
     </div>
     <script src="assets/js/ck_editor.js"></script>
+    <script id="dsq-count-scr" src="//reverie-1.disqus.com/count.js"  ></script>
```

Comment:
I feel a lot of unwanted changes are there in this PR.. can you please remove the files changes which are not required as a part of this change?